.travis.yml

```diff
@@ -24,7 +24,8 @@ matrix:
         - TEST_ASCII="true"
         # space separated list of optional dependencies(conda packages) to install and test
         # autowrap installs libgfortran libgcc gcc cython
-        - TEST_OPT_DEPENDENCY="numpy scipy gmpy2 matplotlib=2.1.0 theano llvmlite autowrap python-symengine=0.3.* tensorflow numexpr ipython"
+
+        - TEST_OPT_DEPENDENCY="numpy scipy gmpy2 matplotlib=2.1.0 theano llvmlite autowrap python-symengine=0.3.* tensorflow numexpr ipython antlr-python-runtime>=4.7,<4.8 antlr>=4.7,<4.8"
```

Comment:
how do we feel about this level of pinning for now?

Reply:
I think this is fine for now. When a new version comes out we can figure out how different it is and if it's backwards or forwards compatible.